README.md

```diff
@@ -53,7 +56,9 @@ There is a known issue in WindowsBase that causes crashes when handling large da
 | .NET Standard | NuGet                    | .NET Core 1.0 |
 
 ## How to install the NuGet package?
-The package you want to install is DocumentFormat.OpenXml. See https://dotnet.myget.org/gallery/open-xml-sdk 
+The package you want to install is DocumentFormat.OpenXml. 
+
+See https://www.nuget.org/packages/DocumentFormat.OpenXml for the release packages or https://dotnet.myget.org/gallery/open-xml-sdk for the latest build packages.
```

Comment:
What about:

NuGet packages are available for [release builds](https://www.nuget.org/packages/DocumentFormat.OpenXml) or [CI Builds](https://dotnet.myget.org/gallery/open-xml-sdk) and follow [semantic versioning](http://www.semver.org).